doc/user/content/sql/alter-cluster.md

```diff
@@ -15,8 +15,21 @@ cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
 
 {{< diagram "alter-cluster-reset.svg" >}}
 
+### `with_options`
+
+{{< private-preview />}}
+
+{{< diagram "with-options-alter-cluster-set.svg" >}}
+
 {{% cluster-options %}}
 
+
+### `WITH (WAIT UNTIL READY)` options
```

Comment:
Do we expect people with sources and sinks to use these options as well since they will still experience downtime?  That is, is it better to use in all cases? Or do we want to add a sentence that this is for clusters without sinks and sources? (and link to the example below)?

Reply:
Yeah, unfortunately, for now, this is only allowed on clusters without a sink or source. All resizes on clusters with sinks or sources will have to use the old mechanism which will cause downtime. 

The new mechanism essentially performs a `create before delete` operation, whereas the old mechanism does a `delete before create`. Sinks and Sources must run on only one replica, so we cannot use the new mechanism.  Currently we'll error during this alter in the same way that we'd error if a customer were to adjust the replication factor on a cluster with sources to be greater than 1.